packages/strapi-plugin-content-type-builder/admin/src/containers/FormModal/utils/forms.js

```diff
@@ -751,6 +715,35 @@ const forms = {
           ]);
         }
 
+        if (type === 'uid') {
+          const options = Object.keys(attributes)
```

Comment:
I don't really see why you need to translate the attributes here

Reply:
Maybe I don't understand your comment but the only translation here is for the "None" option.